packages/zowe-explorer/i18n/sample/package.i18n.json

```diff
@@ -77,6 +77,7 @@
   "zowe.commands.history": "Toggle if Commands persist locally",
   "zowe.automaticProfileValidation": "Allow automatic validation of profiles.",
   "zowe.settings.version": "Track if migration has been processed",
+  "zowe.security.enableSecureCredentials": "Store username and password for z/OS as clear text in Zowe team config profiles if disabled",
```

Comment:
This name doesn't match the name used in package.json `zowe.security.secureCredentialsEnabled`

Reply:
Done.